sdk/core/azure-core/azure/core/pipeline/policies/_distributed_tracing.py

```diff
@@ -112,7 +112,15 @@ def end_span(
         response: Optional["HttpResponseType"] = None,
         exc_info: Optional[Tuple] = None,
     ) -> None:
-        """Ends the span that is tracing the network and updates its status."""
+        """Ends the span that is tracing the network and updates its status.
+
+        :param request: The PipelineRequest object
+        :type request: ~azure.core.pipeline.PipelineRequest
```

Comment:
Do we still need the `:type ...` docstrings? Didn't we enable type hint support from sphinx

Reply:
I'm not aware of enabling this. @l0lawrence ?

Reply:
Hmmm.... I know I asked to add [sphinx-autodoc-typehints]( https://pypi.org/project/sphinx-autodoc-typehints) at some point. Doesn't look like that ever happened. Not sure if there was a reason or if it just fell through the cracks (and I missed picking it up again)